readthedocs/projects/tasks.py

```diff
@@ -1517,7 +1517,7 @@ def clean_build(version_pk):
     # because we are syncing the servers with an async task.
     del_dirs = [
         os.path.join(version.project.doc_path, dir_, version.slug)
-        for dir_ in ('checkouts', 'envs', 'conda')
+        for dir_ in ('checkouts', 'envs', 'conda', '.cache')
```

Comment:
We have similar code for wiping versions.
Shouldn't we use that -- https://github.com/readthedocs/readthedocs.org/blob/4b32becbbe40855bf8fb5583057c7a4b4be0bd6a/readthedocs/core/utils/general.py#L12

Also #6186 is making some changes to this function.

Reply:
That function broadcast the task to all builders, here we only want to use it on one builder (the builder we are executing the task)